GIS-war/src/main/java/giswar/listener/StartupListener.java

```diff
@@ -27,6 +27,18 @@ public class StartupListener implements ServletContextListener {
     @Override
     public void contextInitialized(ServletContextEvent sce) {
 
+        for (String property : batchProperties.stringPropertyNames()) {
+            String value = batchProperties.getProperty(property);
+
+            // Check if the property contains a reference to an environment
+            // variable of the form ${ENV=VARIABLE_NAME}
+            value = value.replace(" ", "");
+            if (value.startsWith("${ENV=")) {
```

Comment:
At the top of the class, you could define something like the following,
`public static final String ENV_PREFIX = "${ENV=";`
...then do an `ENV_PREFIX.length()` to replace the magic-ish number of 6 in line 37.